notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1404 @@
+    "         - [GriddedPSFModel](https://photutils.readthedocs.io/en/stable/api/photutils.psf.GriddedPSFModel.html#photutils.psf.GriddedPSFModel) - enables spatially-dependent PSF grids\n",
+    "         - [STDPSFGrid](https://photutils.readthedocs.io/en/stable/api/photutils.psf.STDPSFGrid.html#photutils.psf.STDPSFGrid) - reads STScI standard PSF models into a grid\n",
+    "         - [stdpsf_reader](https://photutils.readthedocs.io/en/stable/api/photutils.psf.stdpsf_reader.html#photutils.psf.stdpsf_reader) - reads STScI standard PSF format models\n",
+    "         - [FittableImageModel](https://photutils.readthedocs.io/en/stable/api/photutils.psf.FittableImageModel.html#photutils.psf.FittableImageModel) - convert stacked stars to fitable model\n",
+    " - [**scipy**](https://scipy.org) - mathematical interpolation and shift functions\n",
+    "     - [ndimage.shift](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.shift.html) - fractional array shifts\n",
+    " - [**drizzlepac**](https://www.stsci.edu/scientific-community/software/drizzlepac) - combine HST images into mosaics\n",
+    "     - [astrodrizzle](https://drizzlepac.readthedocs.io/en/deployment/astrodrizzle.html) - combine exposures by drizzling\n",
+    "     - [tweakreg](https://drizzlepac.readthedocs.io/en/deployment/tweakreg.html) - align exposures to a common WCS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "acfcc04d-596f-41a6-a2fa-bce20ee2cfbf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import glob\n",
+    "import shutil\n",
+    "import tarfile\n",
+    "from IPython.display import clear_output\n",
+    "import numpy\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot\n",
+    "from matplotlib.colors import LogNorm\n",
+    "from matplotlib.patches import Rectangle\n",
+    "from astroquery.mast import Observations\n",
+    "from astropy.io import fits\n",
+    "from astropy.table import Table, QTable\n",
+    "from astropy.visualization import simple_norm\n",
+    "from astropy.stats import sigma_clipped_stats\n",
+    "from astropy.stats import SigmaClip\n",
+    "from photutils.detection import DAOStarFinder\n",
+    "from photutils.aperture import aperture_photometry\n",
+    "from photutils.aperture import CircularAperture\n",
+    "from photutils.aperture import CircularAnnulus\n",
+    "from photutils.aperture import ApertureStats\n",
+    "from photutils.psf import PSFPhotometry\n",
+    "from photutils.psf import SourceGrouper\n",
+    "from photutils.psf import IntegratedGaussianPRF\n",
+    "from photutils.psf import GriddedPSFModel\n",
+    "from photutils.psf import STDPSFGrid\n",
+    "from photutils.psf import FittableImageModel\n",
+    "from drizzlepac import tweakreg\n",
+    "from drizzlepac import astrodrizzle\n",
+    "\n",
+    "# Custom functions written in psf_utilities.py.\n",
```

Comment:
I agree this would condense the import statements, but then requires calling e.g. `psf_utilities.create_mask`, which lengthens the function calls throughout the full notebook?

Reply:
True, might be easier to just keep as is 👍🏾